### PR TITLE
Fixed a bug in StandardSynchronizer.cs

### PR DESCRIPTION
**Pull request by alobo**
DirectoryInfo.GetFiles() throws an error when it reaches a file with an illegal character in its name. I replaced it with a recursive file function (FileRecursive.cs) - by handling each individual file, the ones with problematic file names can be skipped over.

### CODE REVIEW

File: FileRecursive.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
```
Comment (alobo) [inline FileRecursive.cs:0]: I obtained this class from: http://www.dotnetperls.com/recursively-find-files and I made some modifications to it.

It works perfectly, and is free to use in any projects, whether they be commercial/personal/open source

**Comment by jaranflaath** [comment]
Thanks for your contribution. Will review and test as soon as possible!

**Comment by jaranflaath** [comment]
Thanks for your contribution! A new version including this fix will be released shortly.

**Comment by alobo** [comment]
No problem - I was glad to help! f you need any other assistance on the project, let me know, as I'd love to help out.

**Pull request merged**